packages/pass-style/src/passStyleOf.js

```diff
@@ -359,11 +359,11 @@ export const toThrowable = specimen => {
   if (isErrorLike(specimen)) {
     return toPassableError(/** @type {Error} */ (specimen));
   }
-  // Note that this step will fail if `specimen` would be a passable container
-  // except that it contains non-passable errors that could be converted.
-  // This will need to be fixed to do the TODO above.
-  const passStyle = passStyleOf(specimen);
-  if (isObject(specimen)) {
+  if (!isPrimitive(specimen)) {
+    // Note that this step will fail if `specimen` would be a passable container
+    // except that it contains non-passable errors that could be converted.
+    // This will need to be fixed to do the TODO above.
+    const passStyle = /** @type {PassStyle} */ (passStyleOf(specimen));
```

Comment:
Moving the call to `passStyleOf` inside the conditional is a drive-by. But typescript's imprecision about the negation of an `is` result type provoked the extra case.